src/task/models/mergeTilesTaskBuilder.ts

```diff
@@ -139,45 +142,46 @@ export class MergeTilesTaskBuilder {
     partData.forEach((part: PolygonPart) => {
       const partLayers = this.createPartLayers(part, inputFiles);
       partsLayers.push(...partLayers);
-
-      const currentZoom = degreesPerPixelToZoomLevel(part.resolutionDegree ?? 0);
-      maxZoom = Math.max(maxZoom, currentZoom);
+      maxZoom = Math.max(maxZoom, degreesPerPixelToZoomLevel(part.resolutionDegree ?? 0));
```

Comment:
need a fix after mc-models-types will be fixed, partData.resolutionDegree is a required field

Reply:
fixed